moja-kuhinjica-frontend/src/pages/mealReservation/index.tsx

```diff
@@ -40,11 +41,13 @@ import { routes } from '../../constants/constants'
 const ORDERING = 'ordering'
 const HEADER_TYPE = 'red'
 const INITIAL_MEAL_AMOUNT = 1
+const DISABLED_MESSAGE = 'Ne možete da rezervišete posle 10 ujutru'
 const RESERVATION_SUCCESS = 'Rezervacija je uspešna'
 const RESERVATION_FAIL = 'Neuspešna rezervacija'
 const RESERVATION_SUCCESS_MESSAGE =
     'Vaša rezervacija je sačuvana. Možete je pogledati na stranici'
-const EMPTY_CART_MESSAGE = 'Vaša korpa je prazna, ukoliko želite da pogledate svoje rezervacije možete otići na stranicu'
+const EMPTY_CART_MESSAGE =
+    'Vaša korpa je prazna, ukoliko želite da pogledate svoje rezervacije možete otići na stranicu'
```

Comment:
lets create a file in constants folder that has this stuff in it. And just import from there which ones u need, instead of declaring them all here